web/static/js/components/respondents/RespondentIndex.jsx

```diff
@@ -453,15 +449,63 @@ class RespondentIndex extends Component<Props, State> {
     )
   }
 
+  renderHeader({ displayText, type, key, sortable, dataType }) {
+    const { sortBy, sortAsc } = this.props
+    let className = classNames({
+      thNumber: dataType === 'number',
+      thDate: dataType === 'date'
+    })
+
+    if (sortable) {
+      return (
+        <SortableHeader
+          text={displayText}
+          property={key}
+          sortBy={sortBy}
+          sortAsc={sortAsc}
+          onClick={() => this.sortBy(key)}
+          key={fieldUniqueKey(type, key)}
+          className={className}
+        />
+      )
+    } else {
+      return (
+        <th className={className} key={fieldUniqueKey(type, key)}>
+          {displayText}
+        </th>
+      )
+    }
+  }
+
+  toogleColumn(type, key) {
+    this.props.actions.toogleSelectedColumn(type, key)
+  }
+
+  columnsMenuOption({ type, key, displayText, checked }) {
+    const { selectedFields } = this.props
+    const inputId = `toggle_column_${fieldUniqueKey(type, key)}`
+    return (
+      <DropdownDropdownCheckboxItem
+        key={Math.random().toString()}
+        id={inputId}
```

Comment:
I think is a code small to pass the id for the input here. See comment in DropdownDropdownCheckboxItem

Reply:
Removed! Thanks!!